crate/server_database/src/core/main_db_params.rs

```diff
@@ -53,13 +52,8 @@ impl Display for MainDbParams {
             Self::Postgres(url) => write!(f, "postgres: {}", redact_url(url)),
             Self::Mysql(url) => write!(f, "mysql: {}", redact_url(url)),
             #[cfg(feature = "non-fips")]
-            Self::RedisFindex(url, _, label) => {
-                write!(
-                    f,
-                    "redis-findex: {}, master key: [****], Findex label: 0x{}",
-                    redact_url(url),
-                    hex::encode(label)
-                )
+            Self::RedisFindex(url, _, _) => {
+                write!(f, "redis-findex: {}, master key: [****]", redact_url(url),)
```

Comment:
Do you not display the label when provided? It is now optional but is still here.